docker/images/kuma_base/Dockerfile

```diff
@@ -98,7 +98,9 @@ RUN useradd --uid 1000 --shell /bin/bash --create-home kuma \
 # install Python libraries
 WORKDIR /app
 COPY --chown=kuma:kuma ./requirements /app/requirements
-RUN pip install --no-cache-dir -r requirements/dev.txt
+# RUN pip install --no-cache-dir -r requirements/dev.txt
+# Make it less noisy. Might remove this once Travis and docker-compose is working spiffily
+RUN pip install --no-cache-dir -r requirements/dev.txt --quiet
```

Comment:
Once you're done debugging, I would prefer that `--quiet` was removed. I like to see the details sometimes.

Reply:
On second thought, `--quiet` is fine.